src/Controller/EnMarche/BoardController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+
+/**
+ * @Route("/espace-membres-conseil")
```

Comment:
What about `espace-membre-du-conseil` ?
In application the name of rest of spaces is singular.

Reply:
Marine said on clubHouse Card:
URLs
intro : /espace-membres-conseil
recherche : /espace-membres-conseil/recherche
profils sauvegardes : /espace-membres-conseil/profils-sauvegardes